README.md

```diff
@@ -1,17 +1,26 @@
 # Demo
-This project contains the source of the demo project that is shipped with Scalable Pixel Streaming
+This repository contains the source of the demo UE project that is shipped with Scalable Pixel Streaming.
```

Comment:
Can we add here:
1) What the project is (e.g. vehicle demo? whatever the official language is around it),
2) How its used in SPS
3) Why someone might want to follow the steps below to deploy.

Reply:
Addressed this in [ea6942b](https://github.com/ScalablePixelStreaming/Demo/pull/1/commits/ea6942b879e388d43cc88959a32a6be2988d4fcc)